src/helpers/colors.ts

```diff
@@ -0,0 +1,25 @@
+export const productColors = {
```

Comment:
Nice idea! 
But even better(and more generic approach) will be to create a `theme.ts` file for all stylings, you may store all necessary css variables there and they always will be by hand.